gateway/ln-gateway/src/lib.rs

```diff
@@ -1165,7 +1165,7 @@ impl Gateway {
     /// - `GatewayConfiguration` is read from the database.
     /// - All cli or environment variables are set such that we can create a
     ///   `GatewayConfiguration`
-    pub async fn get_gateway_configuration(&self) -> Option<GatewayConfiguration> {
+    async fn get_gateway_configuration(&self) -> Option<GatewayConfiguration> {
```

Comment:
Is this something you might want to call if you instantiate the gateway yourself as part of a custom binary?

Reply:
I'd say yes, but right now I don't see us supporting custom binaries for gateways